main.go

```diff
@@ -32,18 +33,20 @@ type config struct {
 }
 
 const (
-	maxCommitBodyLenght = 72
+	trimEnding = "..."
 )
 
-func printLogAndExportEnv(gitCmd git.Git, format, env string, trim bool) error {
+var maxCommitMessageLength int
+
+func printLogAndExportEnv(gitCmd git.Git, format, env string, trimmableVariable bool) error {
 	l, err := output(gitCmd.Log(format))
```

Comment:
pls fix the variable names if you are already touching this function: https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
renamed local variables